main.cpp

```diff
@@ -1,11 +1,14 @@
 #include <QApplication>
+#include <QDebug>
+#include <QDir>
 #include <QQmlApplicationEngine>
 
 int main(int argc, char *argv[])
 {
     QApplication app(argc, argv);
-
```

Comment:
Please keep this line break. It separates QApplication block from QQmlApplicationEngine block.